terraform.tfvars.example

```diff
@@ -1,96 +0,0 @@
-## =============================================================================
```

Comment:
This is a needed document that is part of our distribution with terraform. I don not think I would delete this. This provides end users with an easy template for ORCA deployment variables with information similar to the Cumulus document they distribute in the deploy repo.

Reply:
How is it used? Seems to me like this is a duplicate of orca.tf. deployment-with-cumulus.md only refers to modifying cumulus' terraform.tfvars, no mention of our example file.

Reply:
seems like deployment-with-cumulus.md also has the same information so probably this can go away?